src/frontend/Router.js

```diff
@@ -0,0 +1,46 @@
+const getRouter = () => {
+  const getPathname = () => {
+    const {pathname} = window.location;
+    if (pathname === '/calendar' || pathname === 'statistics') return pathname;
```

Comment:
```suggestion
    if (pathname === '/calendar' || pathname === '/statistics') return pathname;
```

`satistics` `patname`에도 `/`를 추가해야 url(`/statistics`)을 통해 바로 접속할 때 해당 view 로 렌더링 될 것 같아요 
지금은 detail view로 렌더링되고 있습니다!

Reply:
우왕 감사해요 !!!